modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (c) 2010, 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since this is a new file, you might as well use 2018 as the "last modified" year. I'll fix the others when I update the repo in late June / early July.